doc/whats_new/upcoming_changes/sklearn.preprocessing/31227.fix.rst

```diff
@@ -0,0 +1,6 @@
+- Now using ``scipy.stats.yeojohnson`` instead of our own implementation of the Yeo-Johnson transform.
+  Fixed numerical stability (mostly overflows) of the Yeo-Johnson transform with
+  `PowerTransformer(method="yeo-johnson")` when scipy version is `>= 1.12`.
+  Initial PR by :user: `Xuefeng Xu <@xuefeng-xu>` completed by `Mohamed Yaich<@yaichm>`,
+  `Oussama Er-rabie<@eroussama>`, `Mohammed Yaslam Dlimi<@Dlimim>`,
+  `Hamza Zaroual<@HamzaLuffy>`, `Amine Hannoun<@AmineHannoun>` and `Sylvain Marié<@smarie>`.
```

Comment:
I think you need something like this, please double-check the rendering in the changelog by clicking on "check the rendered doc" CI status at the bottom of the PR.

```suggestion
  Initial PR by :user:`Xuefeng Xu <xuefeng-xu>` completed by :user:`Mohamed Yaich <yaichm>`,
  :user:`Oussama Er-rabie <eroussama>`, :user:`Mohammed Yaslam Dlimi<@Dlimim>`,
  :user:`Hamza Zaroual <HamzaLuffy>`, :user:`Amine Hannoun<@AmineHannoun>` and :user:`Sylvain Marié <smarie>`.
```

The current rendering is slightly off, see [this](https://output.circle-artifacts.com/output/job/b5aa8097-f2bf-43bb-8473-b1b80ac4d01d/artifacts/0/doc/whats_new/v1.7.html#sklearn-preprocessing)

Reply:
Thanks, the changelog rendering looks good now see [this](https://output.circle-artifacts.com/output/job/87bd1473-3151-4d47-b276-79c18d16d8c1/artifacts/0/doc/whats_new/v1.7.html#sklearn-preprocessing)